docs/source/Install.rst

```diff
@@ -31,9 +31,9 @@ Once you have installed either Ananconda or miniconda, open your terminal and us
 
    .. code-block:: bash
 
-      conda create -n mapreader python=3.8
+      conda create -n mapreader python=3.9
```

Comment:
Any reason to recommend 3.9? Guess not specifying the python version would default to the latest 3.12 which we think is fine, maybe specify 3.12 though in case 3.13 comes out and we haven't tested that

Reply:
hmm I have no idea why we were suggesting 3.8 vs. more recent versions, maybe because its what we used for MapReader experiments/papers. 
I guess for 3.9 it is the most tested option since its what we are running for all our current MapReader work but will have a think if it would be better to switch straight up to 3.12 or maybe just remove the suggestion for which python version to use in the installation docs.

Reply:
I think have faith that your tests working for 3.12 means all good, if someone finds otherwise they can always open a bug issue

Reply:
I just removed the part where you say which python you want